src/Private/Gui.ps1

```diff
@@ -49,10 +49,24 @@ function Start-PodeGuiRunspace
             [System.Reflection.Assembly]::LoadWithPartialName('PresentationFramework') | Out-Null
             [System.Reflection.Assembly]::LoadWithPartialName('PresentationCore') | Out-Null
 
-            # setup the WPF XAML for the server
-            $gui_browser = "
+            # Check for CefSharp
+            $loadCef = $false
+            if (
+                # This seems to be the safest method to detect if it is loaded or not, feel free to suggest a better one
+                ([AppDomain]::CurrentDomain.GetAssemblies() | Where-Object { $_.FullName.StartsWith("CefSharp.Wpf,") })
+            ) {
+                $loadCef = $true
+            }
+
+            # setup the WPF XAML for the server          
+            # Check for CefSharp and used Chromium based WPF if Modules exists
+            if (
+                $loadCef
+            ){
```

Comment:
Can probably just shrink this to: `if ($loadCef) {`

Reply:
Done.